core/src/main/resources/reference.conf

```diff
@@ -550,6 +566,198 @@ datastax-java-driver {
     token-map.enabled = true
   }
 
+  metrics {
+    # The session-level metrics (all disabled by default).
+    session {
+      enabled = [
+        # The number of nodes to which the driver has at least one active connection (exposed as a
+        # Gauge<Integer>).
+        //connected-nodes,
```

Comment:
Very tiny nit, no space between `//` and `connected-notes` where other metrics have a space.